### PR TITLE
Make updates screen scrollable

### PR DESCRIPTION
**Pull request by pranavmangal**

### CODE REVIEW

File: screens/Updates.js [hunk 1]
```diff
@@ -1,21 +1,28 @@
 import React from "react";
-import { StyleSheet, StatusBar, View } from "react-native";
+import { View, ScrollView } from "react-native";
 import { Appbar } from "react-native-paper";
+import Constants from "expo-constants";
+import { DEFAULT_APPBAR_HEIGHT } from "../node_modules/react-native-paper/src/components/Appbar/Appbar.tsx";
```
Comment (hasali19) [inline screens/Updates.js:5]: This looks like a bad idea tbh.

Comment (pranavmangal) [reply screens/Updates.js:5]: So the ScrollView doesn't let me scroll to the last post unless I pad the bottom of LatestUpdates with [header height + our 16 padding for ScrollView content]. I couldn't find a way to get header height in react-native-paper so I manually calculated it.

In hindsight, these values appear to be constants so this calculation could be avoided in favor of a magic number -> which again doesn't seem like a good idea.

I will try to find another solution.

**Comment by hasali19** [comment]
Could you try this code?
```jsx
import React from "react";
import { View, ScrollView } from "react-native";
import { Appbar } from "react-native-paper";

import LatestUpdates from "../components/LatestUpdates.js";

export default function Question({ navigation }) {
  const fullHeight = { flex: 1 };
  return (
    <View style={fullHeight}>
      <Appbar.Header>
        <Appbar.BackAction onPress={() => navigation.goBack()} />
        <Appbar.Content title="All Updates" />
      </Appbar.Header>
      <View style={fullHeight}>
        <ScrollView contentContainerStyle={{ padding: 16 }}>
          <LatestUpdates />
        </ScrollView>
      </View>
    </View>
  );
}
```

**Comment by pranavmangal** [comment]
This works great. It never occurred to me to set the flex value in the root View.

**Pull request merged**